### PR TITLE
Charts - Improve x-axis ticks

### PR DESCRIPTION
**Pull request by pnavarrc**
This PR improves the generation of tick values for the charts. The charts were using the default ticks, which were not optimal for the timespan of our charts. Instead, I'm forcing ticks every 2 weeks. I also moved the axes to their own module so the logic is centralized and to simplify the code on the charts.

**Updated!**: I updated the logic so it will put ticks every 14 days on desktop and 28 days for mobile, and changed the date format so it's always like **May 31**.

## Desktop

image

### Mobile

image

Can @mikelehen or @chasulin review?

### CODE REVIEW

**Comment by pnavarrc** [comment]
oops, I forgot to put back handling for mobile, will update this evening, thanks @mikelehen !

**Comment by pnavarrc** [comment]
Updated @mikelehen !

File: src/components/Charts/Axis.tsx [hunk 1]
```diff
@@ -1,10 +1,18 @@
 import React from 'react';
-import { timeWeek } from 'd3-time';
+import { timeDay } from 'd3-time';
```
Comment (pnavarrc) [inline src/components/Charts/Axis.tsx:2]: Changed it to use days to make sure that the first tick matches the first day in the range, when using `timeWeek` it was rounding up to the first Sunday after the first date.

**Comment by pnavarrc** [comment]
thanks for the review @mikelehen! I changed it to month divisions for mobile as it used to be, merging!

**Comment by mikelehen** [comment]
Awesome, thank you!  😁

**Pull request merged**